packages/common/src/types.ts

```diff
@@ -117,3 +118,9 @@ export enum AppointmentStatus {
 }
 
 export const MANUAL_DONOR_ID = "manual";
+
+export type CoordinatorUpdate = {
```

Comment:
Maybe change to `CoordinatorUpdateProps` ? The current name implies that this is an action

Reply:
I think `props` implies this is an object passed to a component.

How about `CoordinatorLastChangeDetails`?